ros2bag/ros2bag/verb/record.py

```diff
@@ -167,20 +184,41 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             help='Choose the compression format/algorithm. '
                  'Has no effect if no compression mode is chosen. Default: %(default)s.')
 
+    def _check_necessary_argument(self, args):
+        # One options out of --all, --all-topics, --all-services, --services, topics or --regex
```

Comment:
Comments from MichaelOrlov

```suggestion
        # At least one options out of --all, --all-topics, --all-services, --services, topics or --regex
```

Reply:
Updated